autoscheduler/scraper/tests/commands/scrape_courses_tests.py

```diff
@@ -0,0 +1,240 @@
+                           credit_hours=credit_hours, term=term)
+
+        assert True
+
+    def test_parse_course_fills_instructor_and_meeting(self):
+        """ Tests if parse_course also adds an instructor and meeting to the database"""
+
+        # Arrange
+        instructor_id = "John M. Moore"
+        instructor_email = "jmichael@email.tamu.edu"
+        instructor = Instructor(id=instructor_id, email_address=instructor_email)
+
+        meeting_id = 4972230
+        crn = 12323
+        building = "ZACH"
+        begin_time = datetime.time(13, 50, 0)
+        end_time = datetime.time(14, 40, 0)
+        meeting_type = "LEC"
+        meeting_days = [True, False, True, False, True, False, False]
+        section = Section(id=497223, subject="CSCE", course_num=121, section_num=501,
+                          term_code=0, min_credits=0, current_enrollment=0,
+                          max_enrollment=0, instructor=instructor)
+
+        #Act
+        parse_course(self.csce_section_json)
+
+        # Assert
+        Instructor.objects.get(id=instructor_id, email_address=instructor_email)
+        Meeting.objects.get(id=meeting_id, crn=crn, building=building,
+                            meeting_days=meeting_days, start_time=begin_time,
+                            end_time=end_time, meeting_type=meeting_type, section=section)
+        assert True
+
+    def test_parse_section_handles_alphanumeric_section_num(self):
+        """ Tests if parse_section accepts an alphanumeric section_num """
+
+        # Arrange
+        subject = "ENGL"
+        course_num = "210"
+        section_num = "M99"
+        term_code = 202011
+        min_credits = 3
+        max_enroll = 25
+        curr_enroll = 3
+        section_id = 511984
+
+        # Section model requires an Instructor
+        fake_instructor = Instructor(id="Fake", email_address="a@b.c")
+        fake_instructor.save()
+
+        # Act
+        parse_section(self.engl_section_json, fake_instructor)
+
+        # Assert
+        Section.objects.get(id=section_id, subject=subject, course_num=course_num,
+                            section_num=section_num, term_code=term_code,
+                            current_enrollment=curr_enroll, min_credits=min_credits,
+                            max_enrollment=max_enroll, instructor=fake_instructor)
+
+        assert True
+
+    def test_scrape_courses_can_generate_meeting_time(self):
+        """ Tests that scrape_courses.convert_meeting_time can handle a normal time """
+
+        # Arrange/Act
+        time = convert_meeting_time("1230")
+
+        # Assert
+        self.assertEqual(time, datetime.time(12, 30))
+
+    def test_scrape_courses_handles_null_meeting_time(self):
+        """ Tests that scrape_courses.convert_meeting_time can handle Null times """
+
+        # Arrange/Act
+        time = convert_meeting_time(None)
+
+        # Assert
+        self.assertEqual(time, None)

```

Comment:
So turns out this is the ideal way of doing asserts. I'd honestly like the other tests to use this since this is the right way (just `banner_requests_tests` that isn't using it), but I don't think it's all that necessary to change that file in this PR.

For the `assert true`, I think you can just leave them as is, since `self.assertTrue(True)` gives a `Redundant use of assertTrue with constant value True - pylint(redundant-unittest-assert)`

Reply:
All assertions have been fixed, all `assert True` lines have been removed since they're redundant and tests will pass without them